ios/Sources/MeasureSDK/Swift/MeasureInternal.swift

```diff
@@ -255,27 +281,37 @@ final class MeasureInternal { // swiftlint:disable:this type_body_length
     func startBugReportFlow(takeScreenshot: Bool = true,
                             bugReportConfig: BugReportConfig,
                             attributes: [String: Any]? = nil) {
+        guard isStarted else { return }
+
         let transformedAttributes = transformAttributes(attributes)
         startBugReportFlow(takeScreenshot: takeScreenshot, bugReportConfig: bugReportConfig, attributes: transformedAttributes)
     }
 
     func trackBugReport(description: String,
                         attachments: [MsrAttachment] = [],
                         attributes: [String: Any]? = nil) {
+        guard isStarted else { return }
+
         let transformedAttributes = transformAttributes(attributes)
         trackBugReport(description: description, attachments: attachments, attributes: transformedAttributes)
     }
 
     func trackError(_ error: Error, attributes: [String: AttributeValue]? = nil, collectStackTraces: Bool) {
+        guard isStarted else { return }
+
         userTriggeredEventCollector.trackError(error, attributes: attributes, collectStackTraces: collectStackTraces)
     }
 
     func trackError(_ error: NSError, attributes: [String: Any]? = nil, collectStackTraces: Bool) {
+        guard isStarted else { return }
+
         let transformedAttributes = transformAttributes(attributes)
         userTriggeredEventCollector.trackError(error, attributes: transformedAttributes, collectStackTraces: collectStackTraces)
     }
 
     func getDocumentDirectoryPath() -> String? {
+        guard isStarted else { return nil }
```

Comment:
We can return a valid value here as it's just a convenience function, can be used whether or not the SDK is started.